content/docs/guides/contribute/code-reviews/index.en.md

```diff
@@ -10,4 +10,37 @@ resources:
     byline: "Gunnar Morling / CC-BY-CA 4.0"
 ---
 
+# Life of a pull request
+
+The author of a PR is responsible for its "life". He is responsible for contacting the various parties involved, following the review, responding to comments and correcting the code following review.
+
+The reviewer/maintener undertakes to carry out the review quickly, and is also responsible for closing request changes, check commit history and quickly merge the PR if allowed.
+
+> In the case of a large PR, don't hesitate to ask several reviewers to organize themselves, or even to carry out the review together, reviewer and author.
+
+Here's a suggested workflow. The quotation marks indicate the **necessity** of communication to ensure the smooth flow of PR validation:
+
+```mermaid
+sequenceDiagram
+  actor A as PR author
+  actor R as Reviewer/Maintainer
+  
+  A->>R: ASKS "I need a review"
+  R->>A: ANSWERS ✅ "Ok" or ❌ "not ok"
+  
+  R-->>+A: COMMENTS, asks for changments
+  A-->>R: ANSWERS to comments
+  A-->>R: ANSWERS to requests changes
+  A-->>-R: DOES necessary changes in dedicated commits
```

Comment:
Can you specify that you need to use fixup commits ? The CI automatically blocks fixup commits

Reply:
Yes !